Sources/FluentKit/Query/QueryBuilder+Distinct.swift

```diff
@@ -0,0 +1,33 @@
+extension QueryBuilder {
+    public func distinctCount<Field>(on key: KeyPath<Model, Field>) -> EventLoopFuture<Int>
```

Comment:
What do you think about `count(distinct:)` for this method name?

Reply:
That'd be fine by me. I'll update that 👍